### PR TITLE
Updated docuement for JsonPreprocessor package

### PR DESCRIPTION
**Pull request by namsonx**

### CODE REVIEW

**Review by HolQue — COMMENTED** [review]
Hi Son,

please find my comments as sticky notes in attached JsonPreprocessor_reviewed_QT.pdf
[JsonPreprocessor_reviewed_QT.pdf](https://github.com/test-fullautomation/python-jsonpreprocessor/files/9939257/JsonPreprocessor_reviewed_QT.pdf)

File: packagedoc/additional_docs/Description.rst [hunk 1]
```diff
@@ -12,13 +12,452 @@
...
-TO BE CONTINUED
+How to install
+--------------
 
+Firstly, clone `python-jsonpreprocessor <https://github.com/test-fullautomation/python-jsonpreprocessor>`_ 
+repository to your machine.
 
+Then go to python-jsonpreprocessor, using the 2 common commands below to build or install this package:
+
+.. code-block::
```
Comment (test-fullautomation) [inline packagedoc/additional_docs/Description.rst]: Hi Son,
we don't need to write  here the help for the build command.
The chapter "how to install" should be like e.g. here: https://github.com/test-fullautomation/robotframework-testresultwebapptool
Thank you,
 Thomas

**Comment by test-fullautomation** [comment]
Hi Son,
thank you for your pull request.
Please refer to my comments.
Thank you,
 Thomas

**Comment by test-fullautomation** [comment]
Hi Son,
by when do I get an update here?
Thank you,
 Thomas

**Review by test-fullautomation — APPROVED** [review]
Hi Son,
looks O.K. for me!
Thank you,
 Thomas

**Pull request merged**